client/tests/integration/triggers/by_call_trigger.rs

```diff
@@ -275,6 +279,17 @@ fn unregister_trigger() -> Result<()> {
         id: trigger_id.clone().into(),
     };
     let found_trigger = test_client.request(find_trigger.clone())?;
+    assert!(matches!(found_trigger.action.executable, Some(_)));
+    let found_action = found_trigger.action;
+    let found_trigger = Trigger::new(
+        found_trigger.id,
+        Action::new(
+            found_action.executable.expect("Already checked"),
+            found_action.repeats,
+            found_action.technical_account,
+            found_action.filter,
+        ),
+    );
```

Comment:
Yes, so the trigger returned by `FindTriggerById` has another type than original *trigger*, because of executable. We now don't return WASM, only instructions if there are some. So at the beginning `found_trigger` has `Option<Vec<Instruction>>` as `executable`. We need to unwrap it to be able to compare triggers

Reply:
@mversic , can we close this?